src/operator/leaky_relu.cc

```diff
@@ -22,14 +22,16 @@ Operator *LeakyReLUProp::CreateOperator(Context ctx) const {
 DMLC_REGISTER_PARAMETER(LeakyReLUParam);
 
 MXNET_REGISTER_OP_PROPERTY(LeakyReLU, LeakyReLUProp)
-.describe(R"code(Leaky ReLu activation
+.describe(R"code(Rectified Linear Unit that allows a small gradient when the unit is not active.
```

Comment:
Should only be capitalized if introducing an acronym. Language is weird, change to:

"Leaky rectified linear unit activation function. Has a slope of one when the input is positive and a small positive slope when the input is negative."